snmp/tests/test_profiles.py

```diff
@@ -65,16 +65,29 @@ def test_load_profiles(caplog):
         caplog.clear()
 
 
-def run_profile_check(recording_name):
+def run_profile_check(recording_name, profile=None):
```

Comment:
(Nit) For consistency with the rest of the code we could use `profile_name`/`profile` instead of `profile` / `prof_def`.

Reply:
Sure, updated!